src/fal/telemetry/telemetry.py

```diff
@@ -112,6 +114,13 @@ def dbt_installed_version():
     except ImportError:
         return
 
+def get_dbt_adapter_type() -> str | None:
```

Comment:
Can you instead receive the `dbt_config` in here like you are doing for adapter? Sometimes `get_dbt_config()` does not work

for example, when using

```
from fal import FalDbt

FalDbt(profiles_dir='...')
```

So we receive `dbt_config` in the `log_api` for those cases.

Reply:
I see! That's a great suggestion, thanks =)